runtime/src/main/java/org/corfudb/runtime/view/workflows/AddNode.java

```diff
@@ -0,0 +1,68 @@
+package org.corfudb.runtime.view.workflows;
+
+import lombok.extern.slf4j.Slf4j;
+import org.corfudb.protocols.wireprotocol.orchestrator.CreateWorkflowResponse;
+import org.corfudb.runtime.CorfuRuntime;
+import org.corfudb.runtime.clients.ManagementClient;
+import org.corfudb.runtime.view.Layout;
+
+import javax.annotation.Nonnull;
+import java.time.Duration;
+import java.util.List;
+import java.util.UUID;
+
+/**
+ *
+ * A workflow request that makes an orchestrator call to add a new node to
+ * the cluster.
+ *
+ * Created by Maithem on 1/19/18.
+ */
+@Slf4j
+public class AddNode extends WorkflowRequest {
```

Comment:
Ok this is more familiar in the java object world!

Reply:
Its more java-like, but code seems a bit verbose, let me know if you have suggestion.